rebuild/test/unit/aggregatePublicKeys.test.ts

```diff
@@ -0,0 +1,24 @@
+import {expect} from "chai";
+import {aggregatePublicKeys, PublicKey} from "../../lib";
+import {makeNapiTestSets} from "../utils";
+
+describe("Aggregate Public Keys", () => {
+  const sets = makeNapiTestSets(10);
+  const keys = sets.map(({publicKey}) => publicKey);
+
+  describe("aggregatePublicKeys()", () => {
+    it("should return the promise of a PublicKey", () => {
+      const agg = aggregatePublicKeys(keys);
+      expect(agg).to.be.instanceOf(PublicKey);
+    });
+    it("should be able to keyValidate PublicKey", () => {
+      const agg = aggregatePublicKeys(keys);
+      expect(agg.keyValidate()).to.be.undefined;
```

Comment:
key validate would through on invalid ? in that case check for not throw

Reply:
It is impossible to deserialize an invalid PublicKey with the `blst` library.  It will throw when creating the point before aggregation.  I added an error check for invalid key though.  Will that work?